src/main/java/org/pesmypetcare/webservice/dao/usermanager/UserDaoImpl.java

```diff
@@ -179,7 +180,11 @@ public void deleteGroupSubscription(String userUid, String groupName, WriteBatch
 
     @Override
     public void saveMessagingToken(String uid, String token) throws DatabaseAccessException, DocumentException {
+        String currentToken = documentAdapter.getStringFromDocument(Path.ofDocument(Collections.users, uid), "FCM");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "FCM" appears 4 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5561436)